tools/codegen/api/lazy.py

```diff
@@ -58,19 +63,36 @@ def process_ir_type(typ: Type) -> Union[BaseCType, VectorCType, OptionalCType, L
         raise AssertionError(f"unrecognized type {repr(typ)}")
 
 
-def isValueType(typ: Union[Type, BaseCType, OptionalCType, ConstRefCType, MutRefCType,
-                           ListCType, ArrayRefCType, ArrayCType, VectorCType, TupleCType]) -> bool:
+def isValueType(typ: CType) -> bool:
```

Comment:
Is this meant to fix https://github.com/pytorch/pytorch/issues/72852? (It looks like `isValueType()` never actually gets called with a normal `Type` if I'm following correctly, which makes this an easy fix)

Reply:
yes, it is (to both)